server/admin_management/api/app/constants.py

```diff
@@ -101,6 +75,50 @@ class PrivilegeChangeTypeEnum(str, Enum):
     REVOKE = "REVOKE"
     UPDATE = "UPDATE"
 
+class SortOrderEnum(str, Enum):
+    ASC = "asc"
+    DESC = "desc"
+
+class DelegatedAdminSortByEnum(str, Enum):
+    # Note: this is not the exact model column name, requires table column mapping.
+    CREATE_DATE = "create_date"  # first one is the default sort field
```

Comment:
Is this how we determine the default sort field? Would it make sense to define a DEFAULT_SORT_COLUMN constant

Reply:
Ah, no, I was only intended to add a comment to say that is being used as default. I removed the confusing comment.
The real default chosen is set at the Schema for the router here:
![image](https://github.com/user-attachments/assets/e8821926-6ca6-42ac-b77a-7e1e9a19404e)